gpu_ocean/SWESimulators/FBL.py

```diff
@@ -311,13 +306,12 @@ class FBL_periodic_boundary:
     def __init__(self, \
                  gpu_ctx, \
                  nx, ny, \
-                 boundary_conditions, asym_ghost_cells, \
+                 boundary_conditions, \
                  block_width=16, block_height=16 ):
 
         self.boundary_conditions = boundary_conditions
-        self.asym_ghost_cells = asym_ghost_cells
-        self.ghostsX = np.int32(self.asym_ghost_cells[1] + self.asym_ghost_cells[3])
-        self.ghostsY = np.int32(self.asym_ghost_cells[0] + self.asym_ghost_cells[2])
+        self.ghostsX = np.int32(0) #self.asym_ghost_cells[1] + self.asym_ghost_cells[3])
```

Comment:
Is this intentional?

Reply:
I saw it as well, and no :) All open boundary conditions must be checked later, as they are not tested during this work. Adding this task to trello